README.md

```diff
@@ -54,3 +54,58 @@ spark.optimizeSpatialAuto(sourceTable, outputTable, outputLocation)
 // optimize with the user defined block size
 spark.optimizeSpatial(sourceTable, outputTable, outputLocation, blockSize = 20097000)
 ```
+
+### Registering Optimizations on Databricks
+
+> This section and approach are based on [docs]](https://sedona.apache.org/setup/databricks/)
+> from Apache Sedona.
+
+Installing and registering spark optimizations can be tricky. As is often the case when
+working with distributed systems, there are some fundamental sequencing issues that are
+important to understand. Roughly, the startup of a databricks cluster looks something like
+this:
+
+1. The JVM process starts with the cluster default classpath
+2. The spark config is initialized (Here's where we want to enable optimizations)
+3. VFS/DBFS user class paths are mounted
+
+The jar which contains classes that are referenced in step 2 isn't available prior to step 3!
+Fortunately, it is possible to set up initialization scripts which run prior to step 1 and
+which we can use on databricks to ensure our classes are available by the time a cluster
+loads its spark config.
+
+First, write a script to DBFS which can be used to copy jars from DBFS to the local filesystem
```

Comment:
Smth more towards: "to resolve the class loading ordering issue we can use the [DataBricks Init script](https://docs.databricks.com/clusters/init-scripts.html), etc etc